.github/workflows/build.yaml

```diff
@@ -0,0 +1,29 @@
+name: build
+on:
+  pull_request:
+    types: [opened, reopened, synchronize, labeled]
+  push:
+    branches:
+      - 'main'
+      - 'v*'
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Start build
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'ok-to-test') }}
```

Comment:
so i think if we want this to run on `main` as well as PRs with the `ok-to-test` label, we might need to update this condition to always run if the branch is `main`, wdyt?